tools/update-tester/src/installer.rs

```diff
@@ -77,6 +90,36 @@ fn get_current_checkout() -> xshell::Result<String> {
     cmd!("git rev-parse --verify HEAD").read()
 }
 
+fn get_pgx_version(cargo_toml_contents: &str) -> String {
+    let mut version = get_field_val(cargo_toml_contents, "pgx").to_string();
+    version.truncate(3); // only care about the major.minor version
+    version
+}
+
+// find a `<field name> = "<field value>"` and extract `<field value>`
+fn get_field_val<'a>(contents: &'a str, field_name: &str) -> &'a str {
```

Comment:
this is a duplicate of [`tools/post-install/src/main.rs:get_field_val`](https://github.com/timescale/timescaledb-toolkit/blob/8183d126e4898d64892ee358b3b48b8851f7abbf/tools/post-install/src/main.rs#L265-L271)

let me know if I should just use that function instead.

Reply:
you can switch it to the ones in [`scripting-utilities`](https://github.com/timescale/timescaledb-toolkit/blob/8183d126e4898d64892ee358b3b48b8851f7abbf/crates/scripting-utilities/control_file_reader/src/lib.rs)

(background: I really wanted `post-install` install to be fast to compile and run b/c it's part of the normal workflow, so when I started writing other scripts that used similar logic I just copy/pasted the code. However _this_ code is small enough and important enough that I figure it's worth having one crate that defines the canonical layout. However, I never had the chance to go back and change the others 😓 )

Reply:
Oh, nevermind, I was wrong see https://github.com/timescale/timescaledb-toolkit/pull/410#discussion_r866934570